tests/nimble_l2cap/main.c

```diff
@@ -32,6 +32,7 @@
 #include "thread.h"
 #include "thread_flags.h"
 #include "net/bluetil/ad.h"
+#include "xtimer.h"
```

Comment:
Seems like a shame to keep xtimer here... but I understand that this is unrelated

Reply:
Exactly. Also for this use case it does not really matter, as energy consumption is not really a the focus of this test :-)